app/components/Navbar/index.js

```diff
@@ -15,26 +16,20 @@ export const Navbar = () => {
   const handleClose = () => setIsOpen(false);
 
   return (
-    <div className="relative flex justify-between items-center px-8 z-[999] md:max-w-[1536px] mx-auto">
-      <Link href="/">
-        <Image
-          src="/img/home/beta_logo.svg"
-          alt="Etherlink Beta Logo"
-          width={200}
-          height={200}
-          className="mr-2 -ml-4"
-        />
-      </Link>
-      <div className="hidden md:flex items-center gap-6">
-        <div className="flex items-center pr-4 border-r border-[#414349B2] h-[30px] gap-3">
+    <Container className="relative flex justify-between items-center pt-8 lg:pt-[66px] z-[999] md:max-w-[1536px]">
```

Comment:
How come z index 999? It's good practice to make it just as high as you need it  to avoid future issues.